src/main/groovy/com/bmuschko/gradle/docker/tasks/image/Dockerfile.groovy

```diff
@@ -18,6 +18,7 @@ package com.bmuschko.gradle.docker.tasks.image
 import org.gradle.api.DefaultTask
 import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.TaskAction
+import org.gradle.internal.impldep.com.google.common.collect.Maps
```

Comment:
I don't see this import being used anywhere. Further more I think we would want to shy away from using anything internal to gradles API.